src/TropicalGeometry/hypersurface.jl

```diff
@@ -260,3 +260,72 @@ end
 #     @req has_attribute(TropH,:tropical_semiring_map) "no tropical semiring map cached"
 #     return get_attribute(TropH,:tropical_semiring_map)
 # end
+#
+
+@doc raw"""
+   tropical_prevariety(F::Vector{MPolyRingElem},nu::TropicalSemiringMap)
+
+Return the tropical prevariety generated by intersecting tropical hypersurfaces corresponding to elements of `F`.
+
+If `F` is a finite collection of polynomials with coefficients from a given field, return the tropical prevariety obtained by tropicalizing elements of `F` with respect to a given tropicalization map `nu`.
+
+If no `nu` is given, default to trivial valuation with `min` convention.
+
+If `F` is a collection of tropical polynomials, the function computes and intersects the associated hypersurfaces.
+# Example
+We compute the Dressian $\text{Dr}(2,5)$ below.
+```jldoctest
+julia> Gr25 = grassmann_pluecker_ideal(2,5)
+Ideal generated by
+  x[[1, 2]]*x[[3, 4]] - x[[1, 3]]*x[[2, 4]] + x[[1, 4]]*x[[2, 3]]
+  x[[1, 2]]*x[[3, 5]] - x[[1, 3]]*x[[2, 5]] + x[[1, 5]]*x[[2, 3]]
+  x[[1, 2]]*x[[4, 5]] - x[[1, 4]]*x[[2, 5]] + x[[1, 5]]*x[[2, 4]]
+  x[[1, 3]]*x[[4, 5]] - x[[1, 4]]*x[[3, 5]] + x[[1, 5]]*x[[3, 4]]
+  x[[2, 3]]*x[[4, 5]] - x[[2, 4]]*x[[3, 5]] + x[[2, 5]]*x[[3, 4]]
+
+#Compute Dressian without specified tropicalization map.
+julia> Dr25 = tropical_prevariety(gens(Gr25))
+Polyhedral complex in ambient dimension 10
+
+julia> rays_modulo_lineality(Dr25)[1]
+10-element SubObjectIterator{RayVector{QQFieldElem}}:
+ [1, -1//3, -1//3, -1//3, -1//3, -1//3, -1//3, 1//3, 1//3, 1//3]
+ [1, 1, -1, -1, 1, -1, -1, -1, -1, 3]
+ [1, -1, 1, -1, -1, 1, -1, -1, 3, -1]
+ [1, -1, -1, 1, -1, -1, 1, 3, -1, -1]
+ [-1, 1, 1, -1, -1, -1, 3, 1, -1, -1]
+ [-1, -1, -1, 3, 1, 1, -1, 1, -1, -1]
+ [-1, 1, -1, 1, -1, 3, -1, -1, 1, -1]
+ [-1, 3, -1, -1, -1, 1, 1, -1, -1, 1]
+ [-1, -1, 3, -1, 1, -1, 1, -1, 1, -1]
+ [-1, -1, 1, 1, 3, -1, -1, -1, -1, 1]
+
+#Compute with respect to max convention
+julia> nu = tropical_semiring_map(QQ,max)
+Map into Max tropical semiring encoding the trivial valuation on Rational field
+
+julia> Dr25max = tropical_prevariety(gens(Gr25),nu)
+Polyhedral complex in ambient dimension 10
+
+#Give input as tropical polynomials
+Dr25 = tropical_prevariety(tropical_polynomial.([f for f in gens(Gr25)]))
+Polyhedral complex in ambient dimension 10
```

Comment:
```suggestion
julia> Dr25 = tropical_prevariety(tropical_polynomial.([f for f in gens(Gr25)])) # Give input as tropical polynomials
Polyhedral complex in ambient dimension 10
```
There is a `julia> ` missing, also I don't think comments like this before the `julia>` lines will work, we do have it like this in other cases so I added this as a suggestion:
```julia
julia> some + code   # some explanation
```